codec-http/src/main/java/io/netty/handler/codec/http/HttpObjectEncoder.java

```diff
@@ -109,10 +110,12 @@ protected void encode(ChannelHandlerContext ctx, Object msg, List<Object> out) t
         //     ch.write(Unpooled.EMPTY_BUFFER).addListener(ChannelFutureListener.CLOSE);
         //
         // See https://github.com/netty/netty/issues/2983 for more information.
```

Comment:
Should this comment be reworded? It's no more an `EMPTY_BUFFER` related issue but one more generally related to non-readable buffers.

Reply:
I think its OK for the comment to reference EMPTY_BUFFER as the "typical example" however the code also supports any empty buffer, and this PR doesn't change that behavior. The next question is do we want/need to continue to support this behavior which is less clear.